editor/src/messages/tool/tool_messages/artboard_tool.rs

```diff
@@ -196,6 +214,39 @@ impl ArtboardToolData {
 			dimensions: size.round().as_ivec2(),
 		});
 
+		let top_edge_resized = bounds.selected_edges.as_ref().map_or(false, |edges| edges.top);
+		let left_edge_resized = bounds.selected_edges.as_ref().map_or(false, |edges| edges.left);
+		let original_left_edge_x = bounds.bounds[0].x;
+		let original_top_edge_y = bounds.bounds[0].y;
```

Comment:
Since you are basing the shift on the viewport coordinates, it doesn't work well with the auto panning.